.github/workflows/pr-validation.yml

```diff
@@ -56,6 +56,5 @@ jobs:
       - name: Check license files
         run: dev/check-all-licenses
 
-      # keeping on gradle, `mvn site javadoc:javadoc`, to figure out later
       - name: Generate Javadoc
-        run: ./gradlew generateApiJavadoc
+        run: mvn clean -B -nsu -am -pl :bookkeeper-common,:bookkeeper-server,:bookkeeper-stats-api,:bookkeeper-stats-providers,:codahale-metrics-provider,:prometheus-metrics-provider install javadoc:aggregate -DskipTests -Pdelombok
```

Comment:
the Maven standard syntax for `-pl` does not require the colon, I'd prefer to remove them